plugins/acss-kit/assets/foundation/sass/_globals.scss

```
@@ -0,0 +1,166 @@
+  font-weight: var(--fs-weight);
+  margin-block-start: 0;
+  margin-block-end: 1rem;
+  line-height: 1.2;
+}
+
+hr {
+  --hr-border-width: 0.0625rem;
+  --hr-border-color: var(--color-border-subtle);
+  --hr-margin-block: 1.5rem;
+  --hr-border-style: solid;
+  border: none;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)